spec/jobs/breaker_status_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe BreakerStatus do
+  describe '#perform' do
+    let(:service) do
+      Breakers.client.services.first
+    end
+
+    before(:each) do
+      # Reset breakers before each test
+      Breakers.client.redis_connection.redis.flushdb
+    end
+
+    after(:all) do
+      Breakers.client.redis_connection.redis.flushdb
+    end
+
+    context 'no failures on test service' do
+      it 'reports up to statsd' do
+        expect { subject.perform }.to trigger_statsd_gauge("api.external_service.#{service.name}.up", value: 1)
```

Comment:
Minor nit (not blocking) - may be worth breaking out the gauge name

Reply:
good point, fixed.